roborazzi/src/test/java/com/github/takahirom/roborazzi/GenerateQualifiersTest.kt

```diff
@@ -1,133 +1,146 @@
 package com.github.takahirom.roborazzi
 
-import java.io.File
-import javax.xml.parsers.DocumentBuilderFactory
-import kotlin.math.roundToInt
+import kotlinx.coroutines.runBlocking
+import kotlinx.serialization.decodeFromString
+import nl.adaptivity.xmlutil.serialization.DefaultXmlSerializationPolicy
+import nl.adaptivity.xmlutil.serialization.XML
+import nl.adaptivity.xmlutil.serialization.XmlConfig.Companion.IGNORING_UNKNOWN_CHILD_HANDLER
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.executeAsync
+import okio.ByteString.Companion.readByteString
+import okio.FileSystem
+import okio.Path.Companion.toPath
+import org.apache.commons.compress.archivers.tar.TarArchiveEntry
+import org.apache.commons.compress.archivers.tar.TarArchiveInputStream
+import org.apache.commons.compress.compressors.gzip.GzipCompressorInputStream
 import org.junit.Ignore
 import org.junit.Test
-import org.w3c.dom.Document
 import org.w3c.dom.Node
 import org.w3c.dom.NodeList
 
+
 class GenerateQualifiersTest {
-  /**
-   * Before execute this. Please run `scripts/download_device_xml.sh`
-   */
+  val devicesTarGz =
+    ("https://android.googlesource.com" +
+      "/platform/tools/base/+archive/refs/heads/mirror-goog-studio-main/sdklib/" +
+      "src/main/resources/com/android/sdklib/devices.tar.gz").toHttpUrl()
+
+  val client = OkHttpClient()
+
+  val xml = XML {
+    policy = DefaultXmlSerializationPolicy(
+      pedantic = false,
+      unknownChildHandler = IGNORING_UNKNOWN_CHILD_HANDLER
+    )
+    repairNamespaces = true
+  }
+
   @Test
   @Ignore
```

Comment:
Now we are going to use fields like client and xml for tests. So, I think it is better to use @Ignore for the class. 👀 

```kotlin
@Ignore
class GenerateQualifiersTest {
```